meshtastic/interfaces/ble/connection.py

```
@@ -273,18 +502,54 @@ def _update_client_reference(
+            try:
+                close_thread = self._thread_create_thread(
+                    target=self._safe_close_client,
+                    args=(client_to_close,),
+                    name="BLEClientClose",
+                    daemon=True,
+                )
+                self._thread_start_thread(close_thread)
+                thread_ident, thread_is_alive = _thread_start_probe(close_thread)
+                if thread_ident is None and not thread_is_alive:
+                    self._safe_close_client(client_to_close)
+            except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)